platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -64,6 +64,24 @@ The new capability addresses a common scenario: resolving files shared across al
 
 Refer to [`ProjectLayout.getSettingsDirectory()`](org/gradle/api/file/ProjectLayout.html#getSettingsDirectory()) for additional details.
 
+#### Configurations are initialized lazily
+
+Similar to [Tasks](userguide/lazy_configuration.html), Configurations are now initialized lazily when necessary.
+
+Starting with this release, applying the `Base` plugin or any of its derived plugins (e.g., JVM plugins) will not cause configurations declared using `register` or using the incubating role-based factory methods to be eagerly realized unless required.
```

Comment:
This should be `base` and possibly link to base_plugin user manual chapter. 

```suggestion
Starting with this release, applying the `base` plugin or any plugin that applies the `base` plugin (e.g., JVM plugins) will no longer realize all configurations declared using `register` or the incubating role-based factory methods. 

In some builds, this will reduce configuration time and memory usage. 
```

Reply:
Do we have any way of quantifying the impact?

Reply:
I don't expect much atm. But, I would expect more when the JVM plugins switch over to registering role-locked configurations 9.0 (vs eagerly creating unlocked configurations)